examples/goconnector/main.go

```diff
@@ -17,25 +16,59 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-func main() {
-	url := strings.Trim(os.Getenv("URL"), " ")
-	if url == "" {
-		panic("missing 'URL' env variable")
+type config struct {
+	// URL of service, e.g. "api-ots-test.integration.oddin.gg:443"
+	URL string `envconfig:"URL" required:"true"`
+
+	// Search for client's OTS (service = Kabal) token in the table `access_tokens`
```

Comment:
This comment sounds like it is for internal knowledge, but this repo is public.

Reply:
Nice catch Thanks. Consider it already updated.